yarn.lock

```diff
@@ -5554,50 +5554,6 @@
     "@webassemblyjs/wast-parser" "1.9.0"
     "@xtuc/long" "4.2.2"
 
-"@wepublish/api-db-mongodb@*":
-  version "2.15.0"
-  resolved "https://registry.npmjs.org/@wepublish/api-db-mongodb/-/api-db-mongodb-2.15.0.tgz#441f6bc6be6f27c70ba42555a5ed0cb8b5de92c1"
-  integrity sha512-FzcpRfu6QTi1HkpHF7/km71fxAdVs8FazDo6m3XFXPsCUMRfHM/YxSKtUAZQI7EafHMczPxHWn8J9C5yFj8h5g==
-  dependencies:
-    bcrypt "^5.0.0"
-    mongodb "^3.5.4"
-    nanoid "^2.1.11"
-
-"@wepublish/api-media-karma@*":
-  version "2.15.0"
```

Comment:
Why do you remove all this packages?

Reply:
thanks for the review. a mistake it was.